pkg/mvtxdaemon/server/service/rater/rater.go

```diff
@@ -95,20 +100,22 @@ func NewRater(ctx context.Context, mv *v1alpha1.MonoVertex, opts ...Option) *Rat
 			},
 			Timeout: time.Second * 1,
 		},
-		log:     logging.FromContext(ctx).Named("Rater"),
-		options: defaultOptions(),
+		log:             logging.FromContext(ctx).Named("Rater"),
+		options:         defaultOptions(),
+		lookBackSeconds: atomic.NewFloat64(float64(mv.Spec.Scale.GetLookbackSeconds())),
```

Comment:
do we need nil check for GetLookbackSeconds?

Reply:
We have one inside the function 
```
func (s Scale) GetLookbackSeconds() int {
	if s.LookbackSeconds != nil {
		return int(*s.LookbackSeconds)
	}
	return DefaultLookbackSeconds
}
```